Feat/unlock design

ahlem-jarrar opened the pull request:
New Tella unlocking design/flow management (PIN-Password-Pattern) + New onboarding design and splash screen . [Figma deign](https://www.figma.com/file/MukZyIiIbffGrrAqaffSWX/Onboarding?node-id=535%3A4)

tomislavr commented on tella-locking-ui/src/main/java/com/hzontal/tella_locking_ui/ui/pattern/PatternUnlockActivity.kt:
```diff
@@ -8,28 +8,28 @@ import com.hzontal.tella_locking_ui.patternlock.ConfirmPatternActivity
 
         TellaKeysUI.getMainKeyStore().load(config.wrapper, PBEKeySpec(mNewPassphrase.toCharArray()), object : IMainKeyLoadCallback {
             override fun onReady(mainKey: MainKey) {
+                TellaKeysUI.getUnlockRegistry().setActiveMethod(this@PatternUnlockActivity, UnlockRegistry.Method.TELLA_PATTERN)
```
This setting of active method should not be here.

tomislavr reviewed (changes requested):
Points on looks:

1. "OK" button in PIN keys is not in design screens.
2. String "Draw your pattern" appear after clearing pattern. Shouldn't the string stay the same "Draw your pattern by connecting at least six dots".
3. "You're all done" screen's image (T) has non-transparent background.

ahlem-jarrar commented:
> Points on looks:
> 
> 1. "OK" button in PIN keys is not in design screens.
> 2. String "Draw your pattern" appear after clearing pattern. Shouldn't the string stay the same "Draw your pattern by connecting at least six dots".
> 3. "You're all done" screen's image (T) has non-transparent background.

1. About PIN button I've dicussed with raph on asana and he says that it's okay to have okay button in SetPinScreeen too.
2. Fixed
3. Fixed
